### PR TITLE
P12

### PR DESCRIPTION
**Pull request by lucasperin**

### CODE REVIEW

**Comment by lucasperin** [comment]
Hi @newpavlov !

**Comment by newpavlov** [comment]
You need to fix the CI failures.

File: p12/src/main.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+fn foo(a: u64) -> u64 { a as u64 }
```
Comment (newpavlov) [inline p12/src/main.rs:1]: Remove this functions to fix the CI.

**Review by newpavlov — CHANGES_REQUESTED** [review]

File: p12/src/main.rs (hunk 1 above)
Comment (lucasperin) [reply p12/src/main.rs]: Done!

**Pull request merged**